app/packages/core/src/components/Sidebar/Entries/GroupEntries.tsx

```diff
@@ -207,7 +209,7 @@ const GroupEntry = React.memo(
                   }
                 }}
               />
-              {hovering && !editing && setValue && (
+              {hovering && !editing && setValue && !disabled && (
```

Comment:
should we disable it instead of hiding it? and show the message passed down? `canModifySidebarGroup.message`